client/auth/controller.js

```diff
@@ -20,16 +22,27 @@ import userFactory from 'lib/user';
 import Main from 'components/main';
 import PulsingDot from 'components/pulsing-dot';
 
-module.exports = {
+export default {
 
-	// Login screen used by the desktop application
-	login: function() {
-		if ( OAuthToken.getToken() ) {
-			page( '/' );
-		} else {
-			ReactDom.render(
-				React.createElement( LoginComponent, {} ),
-				document.getElementById( 'primary' )
+	login: function( context ) {
+		if ( config.isEnabled( 'oauth' ) ) {
+			// hack to keep the blue background for Desktop login
+			document.querySelector( '.layout' ).style.backgroundColor = '#0087be';
```

Comment:
My understanding is that the new `Login` page will be used in different flows, such as Jetpack. Hence this page should be easy to customize. I think using such a hack leads us in the opposite direction.

As Drew mentioned, I don't think we have to use the exact same route. Actually I could see us offering a range of routes depending on the context:
* `/login/default`
* `/login/oauth`
* `/login/jetpack`